modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -7996,6 +8001,46 @@ the program reference label and the observation index is 1.
 """
 scalar ObservationReferenceLabel
 
+"""
+An observation validation problem
+"""
+type ObservationValidation {
+  """
+  The type of validation problem
+  """
+  code: ObservationValidationCode!
```

Comment:
Do we need a separate discriminator for importance of the validation? like warning/errorr?

Reply:
I think we may want that in the future, but everything now would be an error I think. I figured we could add another discriminator when we get things that are other than errors. It would be a non breaking change.

Reply:
What do we plan on doing with the code in the UI, if anything?  Feels like we'll be adding new codes frequently.

Reply:
For now, we'll just use it to display the message for the validation types. Eventually, we may be able to offer "Quick Fixes". A suggestion Rob made was, when we have, in the future, unapproved changes for accepted proposals we could offer "Request Approval". I'm hoping we don't often add new codes, just new messages within existing codes, but we'll see...